src/components/Connection/ExtensionConnection.vue

```diff
@@ -238,7 +251,8 @@ $accounts-number: 7;
   }
   &-accounts {
     height: calc(
-      calc(#{$account-height} + #{$account-margin-bottom}) * #{$accounts-number} - #{$account-margin-bottom}
+      calc(#{$account-height} + #{$account-margin-bottom}) * #{$accounts-number} - #{$account-margin-bottom} + #{$account-margin-bottom} /
+        2
```

Comment:
`#{$account-height} + #{$account-margin-bottom}) * #{$accounts-number} - #{$account-margin-bottom} / 2`

this is just completed calculation result